src/components/events/event/EventPage/EventPage.jsx

```diff
@@ -6,13 +6,13 @@ import {
   VerticalAlignBottomOutlined,
   AimOutlined,
 } from '@ant-design/icons';
-import { Button, Divider, Tag, Space, ConfigProvider } from 'antd';
+import { Button, Divider, Tag, Space } from 'antd';
 import moment from 'moment';
-import { AFCBackend } from '../../../util/utils';
-import PostEvent from './postevent/PostEvent';
-import EventVolunteerList from './volunteer-list/EventVolunteerList';
-import EventPageImage from '../../../assets/img/event-page-banner.png';
-import './eventPage.css';
+import { AFCBackend } from '../../../../util/utils';
+import PostEvent from '../PostEvent/PostEvent';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../PostEvent/PostEvent'.